src/Administration/Resources/app/administration/src/app/component/form/sw-price-field/sw-price-field.html.twig

```diff
@@ -27,9 +27,8 @@
             :disabled="isDisabled"
             :name="grossFieldName"
             v-bind="attributesWithoutListeners"
-            @update:model-value="onPriceGrossChange"
+            @update:model-value="onPriceGrossInputChange"
```

Comment:
We're now using one event handler instead of having both `@update:model-value` and `@input-change`, causing conflicts. Thus, the JS part must be adjusted to this new approach.